### PR TITLE
Update versions to 1.3 release

### PR DESCRIPTION
**Pull request by jjimenezTrimble**

### CODE REVIEW

File: android/MapLayersKotlin/app/build.gradle [hunk 1]
```diff
@@ -40,7 +40,7 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
-    implementation 'com.trimblemaps.mapsdk:maps-android-sdk:1.1.0'
-    implementation 'com.trimblemaps.mapsdk:maps-sdk-account:1.1.0'
-    implementation 'com.trimblemaps.mapsdk:maps-android-core:1.0.0'
+    implementation 'com.trimblemaps.mapsdk:maps-android-sdk:1.3.0'
+    implementation 'com.trimblemaps.mapsdk:maps-sdk-account:1.3.0'
+    implementation 'com.trimblemaps.mapsdk:maps-android-core:1.1.0-9'
```
Comment (jschultzTrimble) [inline android/MapLayersKotlin/app/build.gradle:45]: This looks wrong. I se 1.3.0 https://trimblemaps.jfrog.io/ui/native/android/com/trimblemaps/mapsdk/maps-sdk-core/1.3.0/

**Review by jschultzTrimble — CHANGES_REQUESTED** [review]

File: android/MapLayersKotlin/app/build.gradle (hunk 1 above)
Comment (jjimenezTrimble) [reply android/MapLayersKotlin/app/build.gradle:45]: maps-sdk-core is 1.3.0, but maps-android-core is 1.1.0-9 in both the repo and Steven's comment in [MBSDK-3204](https://jira.trimble.tools/browse/MBSDK-3204)

Comment (jschultzTrimble) [reply android/MapLayersKotlin/app/build.gradle:45]: Okay that isn't right. I asked Steven to update but we should be using 1.3.# across the board

Comment (jschultzTrimble) [reply android/MapLayersKotlin/app/build.gradle:45]: Never mind I messed up my path. https://trimblemaps.jfrog.io/ui/native/android/com/trimblemaps/mapsdk/maps-android-core/ this version is right

**Review by jschultzTrimble — APPROVED** [review]

**Pull request merged**